src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/MvcUiConfigurationProvider.cs

```diff
@@ -5,6 +5,11 @@ namespace VolleyManagement.UI.Infrastructure
 {
     public class MvcUiConfigurationProvider : IConfigurationProvider
     {
+        public string GetGoogleClientId()
+        {
+            throw new System.NotImplementedException();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Implement this method or throw 'NotSupportedException' instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3717)